tools/osbuilder/rootfs-builder/nvidia/nvidia_chroot.sh

```diff
@@ -13,14 +13,53 @@ shopt -s extglob
 run_file_name=$2
 run_fm_file_name=$3
 arch_target=$4
-driver_version="$5"
-driver_type="open"
+nvidia_gpu_stack="$5"
+driver_version=""
+driver_type="-open"
 supported_gpu_devids="/supported-gpu.devids"
 
 APT_INSTALL="apt -o Dpkg::Options::='--force-confdef' -o Dpkg::Options::='--force-confold' -yqq --no-install-recommends install"
 
 export DEBIAN_FRONTEND=noninteractive
 
+is_feature_enabled() {
+	local feature="$1"
+	# Check if feature is in the comma-separated list
+	if [[ ",$nvidia_gpu_stack," == *",$feature,"* ]]; then
+		return 0
+	else
+		return 1
+	fi
+}
+
+set_driver_version_type() {
+	echo "chroot: Setting the correct driver version"
+
+	if [[ ",$nvidia_gpu_stack," == *",latest,"* ]]; then
+		driver_version="latest"
+	elif [[ ",$nvidia_gpu_stack," == *",lts,"* ]]; then
+		driver_version="lts"
+	elif [[ "$nvidia_gpu_stack" =~ version=([^,]+) ]]; then
+		driver_version="${BASH_REMATCH[1]}"
+	else
+		echo "No known driver spec found. Please specify \"latest\", \"lts\", or \"driver=<VERSION>\"."
```

Comment:
It looks the the expected format is `version=<version>`, not `driver=<version>`?

Reply:
Done.